src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -141,7 +145,46 @@ def docker_login_to_registry(login_server):
             "Access to registry was denied. Response code: {}".format(response.status_code))
 
     refresh_token = loads(response.content.decode("utf-8"))["refresh_token"]
+    if only_refresh_token:
+        return refresh_token, ''
 
+    authhost = urlunparse((authurl[0], authurl[1], '/oauth2/token', '', '', ''))
+    if repository is None:
+        scope = 'registry:catalog:*'
+    else:
+        scope = 'repository:' + repository + ':pull'
```

Comment:
I was trying to understand the scope and repository here, does it mean if I issue a show-tags command to my repository, I will only get an access_token that has access to my repository only rather than the entire registry?